jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -274,28 +338,29 @@ var module = module;
      Provides access to the charmstore API.
    */
 
+  var charmstoreAPIVersion = 'v4';
```

Comment:
See above, the previous logic in which the GUI was provided a version from elsewhere (the config) is error prone (users needs to pass the_right_version in the_right_format) and meaningless, as we only support a specific single version anyway.